### ISSUE TITLE
Provide badges in other formats

### ISSUE THREAD

**Issue by elwayman02** [mixonic/ember-community-versions#15]
iframe is not the most desirable format for a badge...would be a lot more user-friendly for people if they could simply embed a png or jpg, much like other badge services (badge.fury.io, for example).  I've actually heard of a few people who found out about this project and then went to a badge service to create a custom badge rather than add a jekyll post to this project, simply because they didn't want to embed an iframe.

**Comment by mixonic** [comment]
Right. An image badge implies dynamic image generation though, and thus a server and a bit more complexity than this MVP embodied. Very happy to talk to someone about making this happen.

Images are also extremely important for Stack Overflow, where someone with moderator rights could add an image to out of date SO posts!

**Comment by elwayman02** [comment]
:+1: I don't have a server or I'd offer to help with this :)

**Comment by sheriffderek** [comment]
I see the positives for consistency with the Ember orange etc, but I could see a situation where I had a specific yellow in my blog styling and the orange could be really clashy. I don't have a solution, but wanted to weigh for those who may have a problem with the iframe and or extra img http requests. I generally have to build new social icon UI etc for these reasons. That being said – I really love the purpose of this project.

**Comment by mixonic** [comment]
The iframe introduces a total of one request. There are far better arguments for using an image than number of requests, we should focus on those.

Style-wise I agree perfection is basically impossible. Until visual clash becomes the blocker on adoption (it definitely is not the blocker today) I'm not very inclined to give it much thought.
